### PR TITLE
Expiration

### PR DESCRIPTION
**Pull request by gmcquillan**
- Merging @skorokithakis' changes.
- Removing test that legacy values still work.
- Bumping Version

### CODE REVIEW

File: brake/backends/cachebe.py [hunk 1]
```diff
@@ -51,25 +52,18 @@ def _keys(self, func_name, request, ip=True, field=None, period=None):
...
+        counters.update(cache.get_many(counters.keys()))
+        for key, value in counters.items():
+            # Handle old values.
             if isinstance(value, tuple):
-                count, _ = value # Futureproofing for expiration values.
+                count, expiration = value
             else:
                 count = value
-
+                expiration = time.time()
```
Comment (skorokithakis) [inline brake/backends/cachebe.py]: Hmm, why is this? This would just make it expire right away, no?

Comment (gmcquillan) [inline brake/backends/cachebe.py]: This is your code: https://github.com/gmcquillan/django-brake/pull/21/files#diff-7e302b28a003003ebe630bd6b7a9b6c0R64

Comment (gmcquillan) [inline brake/backends/cachebe.py]: It'd probably be a negative TTL, which would indeed be treated as immediately expired. We should also guard against setting the timeout to 0, because that would mean that key doesn't ever TTL out.

Comment (skorokithakis) [inline brake/backends/cachebe.py]: Right, it looks like I may have meant `time.time() + period` there. You're right about setting the timeout to 0, changing the line to `time.time() + period` would solve that issue as well.

Comment (gmcquillan) [inline brake/backends/cachebe.py]: Good call. I'll add that and some tests for edge cases like that. Good eye!

**Comment by gmcquillan** [comment]
@skorokithakis if this looks good to you, I'll merge and release.

**Comment by skorokithakis** [comment]
Yep, looks good to me! Thank you!

**Comment by gmcquillan** [comment]
Cool. It's live. Let me know how it goes.

Thanks for the feature idea. It's a good one!

**Comment by skorokithakis** [comment]
Will do, we're deploying it now. Thank you for the help!
